dkist/io/asdf/resources/schemas/dataset-1.1.0.yaml

```diff
@@ -0,0 +1,49 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "asdf://dkist.nso.edu/schemas/dataset-1.1.0"
+
+title: |
+  A DKIST Level 1 Dataset object.
+description:
+  The container for a distributed dataset object.
+
+type: object
+properties:
+  data:
+    tag: "asdf://dkist.nso.edu/tags/file_manager-1.0.0"
+
+  wcs:
+    description: The coordinate system for the complete dataset.
+    anyOf:
+      - tag: "tag:stsci.edu:gwcs/wcs-1.0.0"
+      - tag: "tag:stsci.edu:gwcs/wcs-1.1.0"
+
+  mask:
+    tag: "tag:stsci.edu:asdf/core/ndarray-1.0.0"
+
+  unit:
+    tag: "tag:stsci.edu:asdf/unit/unit-1.0.0"
```

Comment:
....whereas down here we require that all 1.1.0 datasets have units. Is that right?

Reply:
yes, which is a backwards-incompatible change, so a new schema version is required.